amazon-qldb-driver-core/src/driver.rs

```diff
@@ -60,19 +61,32 @@ impl QldbDriverBuilder {
     /// Build a `QldbDriver` using the AWS SDK for Rust.
     ///
     /// `sdk_config` will be used to create a configured instance of the SDK.
-    pub async fn sdk_config(self, sdk_config: Config) -> QldbResult<QldbDriver<QldbSessionSdk>> {
-        let hyper = Client::https();
-        let client = QldbSessionSdk::new(hyper, sdk_config);
+    /// Note that this configuration uses the "sdk config" as opposed to the
+    /// QLDB specific config.
+    pub async fn sdk_config(
+        self,
+        sdk_config: &aws_types::config::Config,
+    ) -> QldbResult<QldbDriver<DynConnector>> {
+        let client = Client::new(sdk_config);
         self.build_with_client(client).await
     }
 
-    pub async fn build_with_client<C>(self, client: C) -> QldbResult<QldbDriver<C>>
+    /// Builds a `QldbDriver` using the AWS SDK for Rust.
+    ///
+    /// Note that `config` is the service-specific (QldbSession) config. For
```

Comment:
Is there a way to alias the import to make that clear in the code? Or is the generic sounding Config type just something to get used to?

Reply:
I'm not sure, this is a newly introduced change in the SDK. I think there might be something clever I can do with generics to make it so there is only 1 API that can accept either.